data/location_hints.txt

```diff
@@ -0,0 +1,303 @@
+Outset Island - Orca - Give 10 Knight's Crests:
+  Type: Sometimes
+  Text: giving a fellow swordsman crests of honor
+Outset Island - Great Fairy:
+  Type: Sometimes
+  Text: visiting the Great Fairy atop Outset
+Outset Island - Jabun's Cave:
+  Type: Sometimes
+  Text: entering the cave at the back of Outset
+Outset Island - Dig up Black Soil:
+  Type: Sometimes
+  Text: teaching a pig to dig for treasure
+
+Windfall Island - Chu Jelly Juice Shop - Give 15 Green Chu Jelly:
+  Type: Sometimes
+  Text: giving a potion master green jelly
+Windfall Island - Chu Jelly Juice Shop - Give 15 Blue Chu Jelly:
+  Type: Always
+  Text: giving a potion master blue jelly
+Windfall Island - Mrs. Marie - Give 40 Joy Pendants:
+  Type: Sometimes
+  Text: giving a schoolteacher forty pendants
+Windfall Island - Lenzo's House - Become Lenzo's Assistant:
+  Type: Sometimes
+  Text: undertaking a photography apprenticeship
+Windfall Island - Lenzo's House - Bring Forest Firefly:
+  Type: Sometimes
+  Text: catching a firefly in a bottle
+Windfall Island - Maggie's Father - Give 20 Skull Necklaces:
+  Type: Sometimes
+  Text: giving a wealthy man twenty necklaces
+Windfall Island - Zunari - Stock Exotic Flower in Zunari's Shop:
+  Type: Sometimes
+  Text: stocking a shop with three flowers
+Windfall Island - Sam - Decorate the Town:
+  Type: Sometimes
+  Text: volunteering to decorate Windfall
```

Comment:
Should this say "a very populated place" (like the island name hints), or some other generic description, instead of "Windfall"?

Reply:
Resolved with 24211463855546dfc64b594f866448dc907ed48f.